ui/app/components/secret-engine/configure-azure.hbs

```diff
@@ -0,0 +1,107 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<form {{on "submit" (perform this.submitForm)}} aria-label="save azure creds" data-test-configure-form>
+
+  <NamespaceReminder @mode="save" @noun="configuration" />
+  <MessageError @errorMessage={{this.errorMessage}} />
+
+  <div class="box is-fullwidth is-sideless">
+    {{! WIF is an enterprise only feature. We default to Azure access type for community users and display only those related form fields. }}
```

Comment:
I was a little confused by this initially. After looking at the component file, this is what I deduced. Feel free to take or leave!
```suggestion
    {{! accessType can be "azure" or "wif" - since WIF is an enterprise only feature we default to "azure" for community users and only display those related form fields. }}
```